libsweep/include/sweep/sweep.h

```diff
@@ -10,12 +10,28 @@
 extern "C" {
 #endif
 
+#if defined _WIN32 || defined __CYGWIN__ || defined __MINGW32__
+// If we are building a dll, set SWEEP_API to export symbols
+#ifdef BUILDING_DLL
```

Comment:
@MikeGitb that is correct. 
@daniel-j-h I switched the naming convention to more closely match the slides you posted, hopefully that clarifies any confusion.